platforms/android/tangram/src/main/java/com/mapzen/tangram/viewholder/GLViewHolderFactory.java

```diff
@@ -5,8 +5,8 @@
 
 public interface GLViewHolderFactory {
     /**
-     * Responsible to create an instance of {@link GLSurfaceViewHolder} or client provided implementation
-     * for {@link GLViewHolder}
+     * Responsible to create an instance of {@link GLSurfaceViewHolder} or {@link TextureViewHolder} or
```

Comment:
Again, it's just a `GLSurfaceView`, no need to call out specific implementations.